styles.css

```diff
@@ -1,3 +1,77 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
-    background-color: lightseagreen;
-}
\ No newline at end of file
+    background: linear-gradient(45deg, rgba(47, 44, 44, 1), rgba(56, 56, 56, 1));
+}
+
+.hero {
+  display: flex;
+  height: 100vh;
+  margin-bottom: 20px;
+}
+
+#description-box {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  color: white;
+  width: 50vw;
+  margin-left: 20px;
+  margin-top: 50px;
+}
+
+#logo {
+  width: 350px;
+  height: 6.5rem;
```

Comment:
Can you go ahead and drop the `height` property from `#logo`? By giving it an explicit height _and_ width, you force the image to be stretched. If you only set one dimension explicitly (like width here), then the height will be `auto`. Meaning it will respect the natural aspect ratio of the image.

I'm toggling on and off this height property here so you can see the difference - 

![Feb-08-2021 08-59-10](https://user-images.githubusercontent.com/13723156/107237411-51c51180-69ec-11eb-9b05-fadc952da636.gif)


For the MVP this is the only thing that needs to change before merging!

Reply:
You are a CSS wizard. Thanks for catching that